src/lines.ts

```diff
@@ -201,11 +201,12 @@ export class Lines extends Base<ILinesSettings> {
     gl.viewport(0, 0, canvas.width, canvas.height);
     gl.viewport(0, 0, canvas.width, canvas.height);
     gl.vertexAttrib1f(this.aPointSize, pointSize);
-    mapMatrix
-      .setSize(canvas.width, canvas.height)
-      .scaleMatrix(scale);
+
     if (zoom > 18) {
-      mapMatrix.translateMatrix(-offset.x, -offset.y);
+      mapMatrix
+        .setSize(canvas.width, canvas.height)
```

Comment:
I believe we still would want to change the size, if we are zoomed to greater than 18 or not.

Reply:
Yes, we change size for each alternative. I just moved setSize() and scaleMatrix() into each branch to reset the matrix in the loops. For zoom > 18 there should be no change.